specification/servicelinker/ServiceLinker.Management/routes.tsp

```diff
@@ -0,0 +1,57 @@
+// FIXME: Operations in this file are not detected as a resource operation, please confirm the conversion result manually
+
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "./models.tsp";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.ResourceManager;
+
+namespace Microsoft.ServiceLinker;
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-interface-requires-decorator" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+interface LinkersOperationGroup {
+  /**
+   * List the dapr configuration supported by Service Connector.
+   */
+  @route("/{+resourceUri}/providers/Microsoft.ServiceLinker/daprConfigurations")
+  @get
+  listDaprConfigurations(
+    ...ApiVersionParameter,
+
+    /**
+     * The fully qualified Azure Resource manager identifier of the resource to be connected.
+     */
+    @path
+    resourceUri: string,
+  ): ArmResponse<DaprConfigurationList> | ErrorResponse;
+}
+
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-interface-requires-decorator" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+interface ConfigurationNamesOperationGroup {
+  /**
+   * Lists the configuration names generated by Service Connector for all target, client types, auth types.
+   */
+  @autoRoute
+  @get
+  @action("configurationNames")
+  list is ArmProviderActionSync<
+    Response = ConfigurationNameResult,
+    Parameters = {
+      /**
+       * OData filter options.
+       */
+      @query("$filter")
+      $filter?: string;
+
+      /**
+       * OData skipToken option for pagination.
+       */
+      @query("$skipToken")
+      $skipToken?: string;
```

Comment:
Nit: sinc eyou already have the proper name in the query decorator you can improve the tsp name
```suggestion
      skipToken?: string;
```

Reply:
If we have tooling doing this, it would be worth a look at improving this